bank-of-anthos/extras/cloudsql-multicluster/README.md

```diff
@@ -10,9 +10,8 @@ Note that in this setup, there is no service communication between the two clust
 
 ## Prerequisites
 
-Install the kubectx command line tool
-
-Anthos license
+- Install the kubectx command line tool
+- Anthos license
```

Comment:
It took way too long for my brain to parse this. "An Anthos license"? An Active Anthos license? Just "Anthos license" made my brain think it was a misformatted section heading talking about the license rather than "you need one"

Reply:
Good callout. Upon reviewing this, I had the same initial confusion! Addressed on the main bank-of-anthos repo: [Minor readability changes](https://github.com/GoogleCloudPlatform/bank-of-anthos/pull/502)